jetty-quic/quic-quiche/quic-quiche-common/src/main/java/org/eclipse/jetty/quic/quiche/Quiche.java

```diff
@@ -119,4 +119,98 @@ static String errToString(long err)
             return "?? " + err;
         }
     }
+
+    // TLS Alerts: https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-6
+    interface tls_alert
```

Comment:
Use an `enum` so you have free `toString()`.

Reply:
That's not making the implementation of `errToString()` any simpler as the mapping from `long` to `String` doesn't come any easier with enums.